.github/workflows/ci.yml

```diff
@@ -64,3 +72,12 @@ jobs:
 
     - name: Build
       run: make -j$(nproc)
+
+    - name: Smoketest
+      # Disassemble NOP
+      run: M=1 ./src/mishegos/mishegos ./workers.spec <<< "90"
```

Comment:
Nit: `> /dev/stderr` here, otherwise (IIRC) we'll be dumping the raw binary fuzzer outputs onto the CI's logs I think?

Reply:
I think we want to keep `stderr` and process `stdout` with `mish2jsonl`